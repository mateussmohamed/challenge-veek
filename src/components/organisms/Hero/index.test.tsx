import React from 'react'

import { Hero } from './index'
import withThemeProvider from '../../theme/withThemeProvider'

describe('[<Hero />]', () => {
  test('matches to snapshot', () => {
    const wrapper = withThemeProvider(() => <Hero />)

    expect(wrapper).toMatchSnapshot()
  })
})
