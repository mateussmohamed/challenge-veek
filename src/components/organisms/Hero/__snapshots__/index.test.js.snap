// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Hero />] matches to snapshot 1`] = `
<FullContainer>
  <HeroContent>
    <Typography
      variant="title"
    >
      Sua Operadora Digital
    </Typography>
    <Typography
      variant="subtitle"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae tempus odio. Nam gravida diam eros, non ultricies velit varius ut.
    </Typography>
  </HeroContent>
  <ScrollDown
    to={
      Object {
        "current": null,
      }
    }
  />
</FullContainer>
`;
